pkgs/applications/office/libreoffice/default.nix

```diff
@@ -241,9 +243,10 @@ in stdenv.mkDerivation rec {
       libxshmfence libatomic_ops graphite2 harfbuzz
       librevenge libe-book libmwaw glm glew
       libodfgen CoinMP librdf_rasqal defaultIconTheme makeWrapper
-    ];
+    ]
+    ++ lib.optional kdeIntegration KDE4DIR;
```

Comment:
`KDE4DIR` was an existing variable that pointed to `kde4.kdelibs`, but was used nowhere. We can delete it and use the value directly if you want.